apps/crn-server/src/data-providers/reminders.data-provider.ts

```diff
@@ -43,25 +75,38 @@ export class ReminderSquidexDataProvider implements ReminderDataProvider {
 
   async fetch(options: FetchRemindersOptions): Promise<ListReminderDataObject> {
     const researchOutputFilter = getResearchOutputFilter();
+    const researchOutputDraftFilter = getResearchOutputDraftFilter();
 
     const {
       findUsersContent,
       queryResearchOutputsContents,
+      draftResearchOutputs,
       queryEventsContents,
     } = await this.squidexGraphqlClient.request<
       FetchReminderDataQuery,
       FetchReminderDataQueryVariables
-    >(FETCH_REMINDER_DATA, {
-      researchOutputFilter,
-      eventFilter: getEventFilter(options.timezone),
-      userId: options.userId,
-    });
+    >(
+      FETCH_REMINDER_DATA,
+      {
+        researchOutputFilter,
+        researchOutputDraftFilter,
+        eventFilter: getEventFilter(options.timezone),
+        userId: options.userId,
+      },
+      { includeDrafts: true },
```

Comment:
isn't this going to affect the existing queries? they might all now start receiving drafts, maybe you should add the status filters into the other ones? I would write an integration test for the existing research ouput reminders - for instance create an identical test but make sure that the RO that was previously given a reminder is now in a draft state - lets see if it shows up (it should not)